db/seedfiles/translations/en/cca/insured.rb

```diff
@@ -211,7 +211,7 @@
   :'en.ssn_not_present' => "SSN not present",
   :'en.tax_covering_applicant_not_present' => "Tax covering applicant not present",
   :'en.information_complete' => "Information Complete",
-  :'en.add_person_to_household' => "Add Person to Household",
+  :'en.add_person_to_household' => "Add Member to Household",
```

Comment:
any concern outright changing the legacy string for non-bs4? I know we talked about this before and we think it's probably ok... just want to confirm

Reply:
I think this string was added recently and is only used in the bs4 section of the FAA pages so it should be safe to change. This change is suggested by https://www.pivotaltracker.com/n/projects/2640062/stories/187863301 as well.